src/app.tsx

```diff
@@ -306,9 +306,6 @@ export default function Chat() {
                                 toolName as keyof typeof tools
                               );
 
-                            // Skip rendering the card in debug mode
```

Comment:
Skipping the card in debug mode is problematic because you can't interact with the tools - The Approve/Reject buttons are on the card, so if you skip rendering it, you can't approve or reject tool calls while debugging.